### PR TITLE
ISSUE_#39 - add configuration manager and manage NET 4.5 web config file

### PR DESCRIPTION
**Pull request by jjarnot-vimanet**

### CODE REVIEW

File: KenticoCloud.Delivery/KenticoCloud.Delivery.csproj [hunk 1]
```diff
@@ -28,9 +28,11 @@
     <GenerateAssemblyVersionAttribute>false</GenerateAssemblyVersionAttribute>
     <GenerateAssemblyFileVersionAttribute>false</GenerateAssemblyFileVersionAttribute>
   </PropertyGroup>
-
+	
   <ItemGroup>
     <PackageReference Include="AngleSharp" Version="0.9.9" />
+    <PackageReference Include="Microsoft.Extensions.Configuration" Version="1.1.2" />
```
Comment (petrsvihlik) [inline KenticoCloud.Delivery/KenticoCloud.Delivery.csproj]: As this is a library, please use as low version numbers as possible (1.0.0) - not to force developers to upgrade their other NuGet dependencies if not necessary.

File: KenticoCloud.Delivery/Configuration/NET45ConfigurationProvider.cs [hunk 2]
```diff
@@ -0,0 +1,26 @@
+﻿using Microsoft.Extensions.Configuration;
+
+#if (NET45)
+using System.Configuration;
+#endif
+
+namespace KenticoCloud.Delivery
+{
+    internal class NET45ConfigurationProvider : ConfigurationProvider, IConfigurationSource
```
Comment (petrsvihlik) [inline KenticoCloud.Delivery/Configuration/NET45ConfigurationProvider.cs]: Let's call it `AppSettingsProvider` (or `ConfigurationManagerProvider`)...whichever you like better.

**Comment by jjarnot-vimanet** [comment]
@petrsvihlik branch: Vimanet:Issue_#39 was updated (ISSUE_#39 - Pull request fixes)
with changes according to your comments.
Thanks!

**Comment by petrsvihlik** [comment]
thanks :)

**Pull request merged**